component/common/loki/client/manager_test.go

```diff
@@ -304,23 +307,3 @@ func TestManager_WALDisabled_MultipleConfigs(t *testing.T) {
 	}
 	require.Len(t, seenEntries, expectedTotalLines)
 }
-
-func TestManager_StopClients(t *testing.T) {
```

Comment:
drive-by: test removed since it was asserting / testing through the internal representation of the manager, which is a smell